ActivitiesAndIntents/app/src/main/res/layout/activity_one.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="net.boondockradio.activitiesandintents.OneActivity">
+
+    <TextView
+        android:id="@+id/text"
```

Comment:
we should try to avoid reusing the same id as much as possible.
if we `refactor -> rename` this will rename all `"@+id/text"` (ask shono/shohei)

`android:id="@+id/txt_activity_one"`